### ISSUE TITLE
public package not found

### ISSUE THREAD

**Issue by tarol** [Hacklone/private-bower#262]

**Comment by tarol** [comment]
i use pm2 start private-bower, and from the log,  the load is successful.
`[32m[bower] [39m [36m18/5/17 04:48:26[39m   Refreshing public packages...
[32m[bower] [39m [36m18/5/17 04:48:27[39m   Cached public packages loaded
[32m[bower] [39m [36m18/5/17 04:48:27[39m   private-bower server started
`
but when i install public package, bower throw error like this
`bower                        ENOTFOUND Package jquery not found`

**Comment by timtosWork** [comment]
Yep, same problem here. Don't know why but suddenly, this error appeared out of nothing. Any updated on this? Any ideas? Thanks for any input!
